README.md

```diff
@@ -53,7 +53,7 @@ Navigate into the extension directory and initialize it as a Git repository:
 
 ```bash
 cd saltext-apache
-git init --initial-branch=main
+git init # If you are not using git < 2.28, it will default to the branch being "master" instead of "main" - you won't be able to change it until after you do a commit.
```

Comment:
**issue (blocking):** let's put this as a note below the commands. recent versions should handle this just fine, so we're really addressing an edge case.

Reply:
sounds good, though I was on git 2.43 and it still doesn't default to naming it to main, but tells you how to. How do you want to deal with that?

Reply:
It won't _default_ to using main, but the command shown before your edit will succeed. We just need a note box below the commands shown here to address the edge case. "If you encounter an error with the init command, it's likely due to..."